client/components/CheckoutForm.js

```diff
@@ -52,7 +52,7 @@ class CheckoutForm extends Component {
     return (
       <div>
         <form className="ui form" onSubmit={this.handleSubmit}>
-          {user.firstName !== 'admin' ? (
+          {user.firstName === 'Guest' ? (
```

Comment:
Change to represent new type field in the User database. Should be checking to see if user.type === 'guest' rather than looking at first name